charts/library/common/templates/classes/_ingress.tpl

```diff
@@ -41,12 +41,13 @@ spec:
   {{- if $values.tls }}
   tls:
     {{- range $values.tls }}
+    {{- $secretName := tpl .secretName $ }}
```

Comment:
I guess we need to verify this is a string before pass it to the template engine.

Reply:
Perhaps something like this?

```suggestion
    {{- $secretName := .secretName | ternary (tpl .secretName $) "" }}
```

Reply:
`ternary` makes stuff so hard to read, I like `default` a bit better 😂
```suggestion
    {{- $secretName := tpl (default "" .secretName) $ }}
```

Reply:
👍🏻 You got it! Changes have been pushed.